packages/core/src/tools.ts

```diff
@@ -295,25 +297,28 @@ export namespace Tools {
 		);
 
 		// Retain default legend additional items
-		if (defaultAdditionalItems && userProvidedAdditionalItems) {
+		if (defaultAdditionalItems && userProvidedAdditionalItems) {	
```

Comment:
Weird extra space here, not sure why prettier didn't catch that ...
```suggestion
		if (defaultAdditionalItems && userProvidedAdditionalItems) {
```